sdks/python/apache_beam/metrics/execution.py

```diff
@@ -203,7 +212,9 @@ def to_runner_api(self):
         [beam_fn_api_pb2.Metrics.User(
             metric_name=k.to_runner_api(),
             distribution_data=v.get_cumulative().to_runner_api())
-         for k, v in self.distributions.items()])
+         for k, v in self.distributions.items()]
+        #TODO(pabloem): Add gauge metrics.
```

Comment:
<!--thread_id:cc_168312643_t; commit:754ea169e823364bac58f08da99eef85944cfd59; resolved:1-->
<!--section:context-quote-->
> **pabloem** wrote:
> Done.

<!--section:body-->
That level of indentation is within the list comprehension, and changing it causes a lint issue. I've left it as it was.